plf-tomcat-resources/src/main/resources/bin/setenv.bat

```diff
@@ -178,4 +178,7 @@ SET CATALINA_OPTS=%CATALINA_OPTS% -Djava.util.Arrays.useLegacyMergeSort=true
 REM # Set the window name
 SET TITLE=eXo Platform ${project.version}
 
+REM # PLF-6965 # set default file encoding to UTF-8 Independently from OS default charset
+SET "CATALINA_OPTS=%CATALINA_OPTS% -Dfile.encoding=UTF-8"
```

Comment:
Please you the same format than others CATALINA_OPTS lines of this file, for consistency (no double quote). And move it under the last CATALINA_OPTS line.